cluster/addons/dns/kube2sky/kube2sky.go

```diff
@@ -57,7 +58,7 @@ const (
 	// Maximum number of attempts to connect to etcd server.
 	maxConnectAttempts = 12
 	// Resync period for the kube controller loop.
-	resyncPeriod = 5 * time.Second
+	resyncPeriod = 30 * time.Minute
```

Comment:
Daniel says this governs ONLY how often we get new full lists.  Let's make it 5 minutes.